src/compiler/commandLineParser.ts

```diff
@@ -3118,20 +3113,21 @@ function parseConfig(
         ownConfig.watchOptions = ownConfig.watchOptions && result.watchOptions ?
             assign(result.watchOptions, ownConfig.watchOptions) :
             ownConfig.watchOptions || result.watchOptions;
-        }
+    }
     return ownConfig;
 
-    function applyExtendedConfig(result: ExtendsResult, extendedConfigPath: string){
+    function applyExtendedConfig(result: ExtendsResult, extendedConfigPath: string) {
         const extendedConfig = getExtendedConfig(sourceFile, extendedConfigPath, host, resolutionStack, errors, extendedConfigCache, result);
         if (extendedConfig && isSuccessfulParsedTsconfig(extendedConfig)) {
             const extendsRaw = extendedConfig.raw;
-            let relativeDifference: string | undefined ;
+            let relativeDifference: string | undefined;
             const setPropertyInResultIfNotUndefined = (propertyName: "include" | "exclude" | "files") => {
                 if (extendsRaw[propertyName]) {
-                    result[propertyName] = map(extendsRaw[propertyName], (path: string) => isRootedDiskPath(path) ? path : combinePaths(
-                        relativeDifference ||= convertToRelativePath(getDirectoryPath(extendedConfigPath), basePath, createGetCanonicalFileName(host.useCaseSensitiveFileNames)),
-                        path
-                    ));
+                    result[propertyName] = map(extendsRaw[propertyName], (path: string) =>
+                        isRootedDiskPath(path) ? path : combinePaths(
+                            relativeDifference ||= convertToRelativePath(getDirectoryPath(extendedConfigPath), basePath, createGetCanonicalFileName(host.useCaseSensitiveFileNames)),
+                            path,
+                        ));

```

Comment:
Having both braces here is confusing for me. I normally visually look for braces and indents to match.

Reply:
This can be manually tweaked to introduce newlines, I believe; this is just the "closest thing" that dprint chose from the current code.

Reply:
Hmmm yeah, this is a little weird in this scenario. I can look into making this one better.